salt/metalk8s/salt/master/files/master_99-metalk8s.conf

```diff
@@ -1,9 +1,33 @@
+{% from "metalk8s/map.jinja" import networks with context %}
+
+{% set ip_candidates = salt.network.ip_addrs(cidr=networks.control_plane) %}
```

Comment:
I personnaly don't like the fact that we do jinja stuff in a configuration file, should be part of the state and added in `context` or `defaults` of `file.managed`

Also that works ? even if `template` is not set to `jinja` in the state ?

Reply:
I believe it works. Also, we're doing the same thing in some other files I believe. Could change though.

Now, given #970, this should be 'fixed' once and for all.

Reply:
It's works indeed but I think in the `file.managed` you need to specify `template: jinja` and that not the case for salt-master